modules/nf-core/fastcov/environment.yml

```diff
@@ -0,0 +1,8 @@
+name: fastcov
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+  - davidfischer
```

Comment:
where is this coming from?

Reply:
@fischer-hub Can you add the package to the bioconda registry?